src/components/ModalLoadingInfo.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+export default function LoadingInfo({ loadingStatus }) {
+  return (
+    <Wrapper>
+      <h2>Ladestatus</h2>
+      <p>Dein Foto wird gerade hochgeladen. Wir bitten Dich um ein wenig Geduld!</p>
+      <p>Lädt:</p>
+      <ProgressBarWrapper loadingStatus={loadingStatus}>
+        <div>
+          <p>{`${loadingStatus}%`}</p>
+        </div>
+      </ProgressBarWrapper>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.article`
+  border: 1px solid #19337a;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  position: absolute;
+  width: 100%;
+  border-radius: 20px;
+  padding: 1rem;
+  gap: 1rem;
+  width: 100%;
+  z-index: 1;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 100%;
+  h2 {
+    font-size: 1.2rem;
+  }
+  p {
+    font-size: 1rem;
```

Comment:
Is this (1rem) necessary here?

Reply:
Thanks Daniel, you´re right...I thought it looks better but actually it doesn´t make any difference...I changed it ;)

Reply:
I think 1rem is the default value